README.md

```diff
@@ -14,25 +14,40 @@ Demonstrates how to use the Blob Storage service.
 Blob storage stores unstructured data such as text, binary data, documents or media files.
 Blobs can be accessed from anywhere in the world via HTTP or HTTPS.
 
-Note: This sample uses the .NET 4.5 asynchronous programming model to demonstrate how to call Azure Storage using asynchronous API calls. When used in real applications, this approach enables you to improve the
+Note: This sample uses the .NET 4.7.2 asynchronous programming model to demonstrate how to call Azure Storage using asynchronous API calls. When used in real applications, this approach enables you to improve the
 responsiveness of your application. Calls to Azure Storage are prefixed by the `await` keyword. For more information about asynchronous programming using the Async/Await pattern, see [Asynchronous Programming with Async and Await (C# and Visual Basic)](https://msdn.microsoft.com/library/hh191443.aspx).
 
 If you don't already have a Microsoft Azure subscription, you can
 get a FREE trial account [here](http://go.microsoft.com/fwlink/?LinkId=330212).
 
 ## Running this sample
 
-This sample can be run using either the Azure Storage Emulator that installs as part of this SDK - or by
+This sample can be run using either the Azurite that installs as part of this SDK - or by
 updating the App.Config file with your AccountName and Key.
-To run the sample using the Storage Emulator (default option):
+To run the sample using the Azurite (default option):
 
-1. Download and Install the Azure Storage Emulator [here](http://azure.microsoft.com/downloads/).
-2. Start the Azure Storage Emulator (once only) by pressing the Start button or the Windows key and searching for it by typing "Azure Storage Emulator". Select it from the list of applications to start it.
-3. Set breakpoints and run the project using F10.
+1. Download and Install the mkcert [here](https://github.com/FiloSottile/mkcert#installation).
+2. Clone Azurite to local [here](https://github.com/Azure/Azurite).
+3. After cloning source code, execute following commands to install Azurite V3.
+
+```bash

```

Comment:
Indent this code block to visually associate it with the list item to which it belongs.